backend/battle/serializers.py

```diff
@@ -19,6 +35,109 @@ class Meta:
         fields = ("id", "battle", "trainer", "pokemons")
 
 
+class TeamCreateSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+
+    battle = serializers.PrimaryKeyRelatedField(queryset=Battle.objects.all())
+    trainer = serializers.PrimaryKeyRelatedField(queryset=User.objects.all())
+
+    pokemon_1 = serializers.CharField(style={'base_template': 'textarea.html'}, required=False)
+    pokemon_2 = serializers.CharField(style={'base_template': 'textarea.html'}, required=False)
+    pokemon_3 = serializers.CharField(style={'base_template': 'textarea.html'}, required=False)
+
+    position_pkn_1 = serializers.ChoiceField(
+        choices=POSITION_CHOICES,
+        label="Position",
+        required=False)
+    position_pkn_2 = serializers.ChoiceField(
+        choices=POSITION_CHOICES,
+        label="Position",
+        required=False)
+    position_pkn_3 = serializers.ChoiceField(
+        choices=POSITION_CHOICES,
+        label="Position",
+        required=False)
```

Comment:
These fields should be required no?

Reply:
true!

Reply:
Actually, if I set true, it returns an error, so I need to set False, and I verify if the field is empty on serializer